docs/api.adoc

```diff
@@ -3,7 +3,7 @@
 :imagesdir: media/
 
 [[api]]
-The react4xp library (link:https://market.enonic.com/vendors/enonic/react4xp-lib[Enonic Market]) exposes a couple of methods that can be run from XP controllers.
+The React4xp library (link:https://market.enonic.com/vendors/enonic/react4xp-lib[Enonic Market]) exposes a couple of methods that can be run from XP controllers.
 
 This chapter also covers the <<#ssr, SSR engine>>, compilation, and <<#react4xp_properties, configuration>> of those.
```

Comment:
> <<#react4xp_properties, configuration>>

Same here - the section is removed and the link is broken.

Reply:
fixed